### PR TITLE
Prepare for beta release

### PR DESCRIPTION
**Pull request by PatrickCronin**
* Migrate the repo to MaxMind's public GitHub account
* Add licenses and a Changelog
* Update the README
* Migrate from codecov.io to coveralls
* Add Goreleaser config

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    replacements:
+      darwin: 'Darwin'
+      linux: 'Linux'
+      windows: 'Windows'
+      386: 'i386'
+      amd64: 'x86_64'
+    vendor: 'MaxMind, Inc.'
+    homepage: 'https://www.maxmind.com/'
+    maintainer: 'MaxMind, Inc. <support@maxmind.com>'
+    description: 'Program to lookup records for one or more IPs/networks in one or more .mmdb databases.'
```
Comment (oalders) [inline .goreleaser.yml]: s/lookup/look up/

File: README.md [hunk 2]
```diff
@@ -1,337 +1,409 @@
-Inspect `.mmdb` databases
+## mmdbinspect
+
+[![Build Status](https://github.com/maxmind/mmdbinspect/workflows/Build/badge.svg)](https://github.com/maxmind/mmdbinspect/actions?query=workflow%3A%22Build%22)
+[![Coverage Status](https://coveralls.io/repos/github/maxmind/mmdbinspect/badge.svg)](https://coveralls.io/github/maxmind/mmdbinspect)
+
+`mmdbinspect` - lookup records for one or more IPs/networks in one or more .mmdb databases
```
Comment (oalders) [inline README.md]: s/lookup/look up/

File: README.md [hunk 3]
```diff
@@ -1,337 +1,409 @@
...
+  db            Path to a MMDB file. Can be specified multiple times.
+  <IP|network>  An IP address, or network in CIDR notation. Can be
+                specified multiple times.
+```
+
+## Description
+
+Any IPs specified will be treated as their single-host network counterparts (e.g. 1.2.3.4 => 1.2.3.4/32).
+
+`mmdbinspect` will lookup each IP/network in each database specified. For each IP/network looked up in a database, the program will select all records for networks which are contained within the looked up IP/network. If no records for contained networks are found in the datafile, the program will select the record that is contained by the looked up IP/network. If no such records are found, none are selected.
```
Comment (oalders) [inline README.md]: s/lookup/look up/

File: README.md [hunk 4]
```diff
@@ -1,337 +1,409 @@
...
+
+## Installation
+
+### Installing via binary release
+
+[Release binaries](https://github.com/maxmind/mmdbinspect/releases) have been made available for several popular platorms. Simply download the binary for your platform and run it.
+
+### Installing on Linux via the tarball
+
+Downlad and extract the appropriate tarball for your system. You will end up with a directory named something like `mmdbinspect_0.0.0_linux_amd64` depending on the version and architecture.
```
Comment (oalders) [inline README.md]: s/Downlad/Download/

File: README.md [hunk 5]
```diff
@@ -1,337 +1,409 @@
...
+
+### Installing on RedHat or CentOS via the rpm
+
+(N.B. You can also use the tarball.)
+
+Run `rpm -i path/to/mmdbinspect_0.0.0_linux_amd64.rpm` (replacing the version number and architecture as necessary). You will need to be root. This will install `mmdbinspect` to `/usr/bin/mmdbinspect`.
+
+### Installing on macOS (darwin) via the tarball
+
+This is the same as installing on Linux visa the tarball, except choose a tarball with "darwin" in the name.
```
Comment (oalders) [inline README.md]: s/visa/via/

File: README.md [hunk 6]
```diff
@@ -1,337 +1,409 @@
...
+
+### Installing on Windows
+
+Download and extract the appropriate zip for your system. You will end up with a directory named something like mmdbinspect_0.0.0_windows_amd64 depending on the version and architecture.
+
+Copy mmdbinspect.exe to where you want it to live.
+
+### Installing from source or Git
+
+_We aim to always support the current and penultimate major releases of the Go compiler. You can get it at the [Go website](https://golang.org)._
```
Comment (oalders) [inline README.md]: s/to always support/always to support/ 

[insert argument about split infinitives]

File: README.md [hunk 7]
```diff
@@ -1,337 +1,409 @@
...
-        "Lookup": "130.113.64.30/24"
-    },
+        "Lookup": "71.160.223.3"
+    }
+]
+```
+</details>
+
+<details>
+    <summary>Lookup one IP/network in multiple databases</summary>
```
Comment (oalders) [inline README.md]: s/Lookup/Look up/

File: README.md [hunk 8]
```diff
@@ -1,337 +1,409 @@
...
+            }
+        ],
+        "Lookup": "202.196.224.4"
+    }
+]
+```
+</details>
+
+<details>
+    <summary>Lookup multiple IPs/networks in a single database</summary>
```
Comment (oalders) [inline README.md]: s/Lookup/Look up/

File: README.md [hunk 9]
```diff
@@ -1,337 +1,409 @@
...
+            }
+        ],
+        "Lookup": "2003::ff00:3a4c/128"
+    }
+]
+```
+</details>
+
+<details>
+    <summary>Lookup multiple IPs/networks in multiple databases</summary>
```
Comment (oalders) [inline README.md]: s/Lookup/Look up/

File: README.md [hunk 10]
```diff
@@ -1,337 +1,409 @@
...
+        ],
+        "Lookup": "1.2.3.66"
     }
 ]
 ```
+</details>
 
-Or, pipe it to `jq`:
+<details>
+    <summary>Lookup a file of IPs/networks using the `xargs` utility</summary>
```
Comment (oalders) [inline README.md]: s/Lookup/Look up/

File: .goreleaser.yml [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+archives:
+  - id: 'mmdbinspect'
+    builds:
+      - 'mmdbinspect'
+    replacements:
+      darwin: 'Darwin'
+      linux: 'Linux'
+      windows: 'Windows'
+      386: 'i386'
+      amd64: 'x86_64'
```
Comment (oschwald) [inline .goreleaser.yml]: I am not sure about these replacements. `geoipupdate` doesn't use them and `amd64` is more common in the Linux world. That said, it looks like `mm-network-analyzer` does use them.

File: .goreleaser.yml [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+    license: 'Apache 2.0 or MIT'
+    formats:
+      - 'deb'
+      - 'rpm'
+    bindir: '/usr/bin'
+    files:
+      'CHANGELOG.md': '/usr/share/doc/mmdbinspect/CHANGELOG.md'
+      'LICENSE-APACHE': '/usr/share/doc/geoipupdate/LICENSE-APACHE'
+      'LICENSE-MIT': '/usr/share/doc/geoipupdate/LICENSE-MIT'
+      'README.md': '/usr/share/doc/geoipupdate/README.md'
```
Comment (oschwald) [inline .goreleaser.yml]: s/geoipupdate/mmdbinspect

File: .goreleaser.yml (hunk 11 above)
Comment (oalders) [reply .goreleaser.yml]: Waiting on a response to this comment before I merge.

File: README.md (hunk 10 above)
Comment (oalders) [reply README.md]: Did this one get resolved?

File: .goreleaser.yml (hunk 11 above)
Comment (PatrickCronin) [reply .goreleaser.yml]: I was cribbing from https://github.com/maxmind/mm-geofeed-verifier/blob/master/.goreleaser.yml. I use CentOS for my servers, and over on our side of the pond `x86_64` is the standard for 64-bit packages (although I realize we're not building any packages specific for CentOS/RedHat). I think there'd be value in making a common formula for all MaxMind releases -- that'd simplify things. I'll take whatever recommendation you have to resolve this.

Comment (oschwald) [reply .goreleaser.yml]: Debian and Ubuntu both use amd64. We should probably standardize our naming and given that `geoipupdate` is the only one of these that has much real usage, I'd suggest just following it for all naming.

**Pull request merged**